### PR TITLE
[docs] Add `other` to licenses

### PR DESCRIPTION
**Pull request by julien-c**
cc @Pierrci

### CODE REVIEW

**Comment by osanseviero** [comment]
What's the use case of this? Doesn't it go against the goal to standardize the licensing? (My worry is people exploiting this one)

**Review by osanseviero — APPROVED** [review]
I see this is in the license picker so approving :)

**Comment by julien-c** [comment]
yes @osanseviero the discussion with @Pierrci was inside [internal PR](https://github.com/huggingface/moon-landing/pull/1642) but maybe buried a little deep so hard to follow

TL;DR: with the new GUI users who would want to assign a license that's not in the list could use "other" and add a `LICENSE` file to their repo). I wouldn't expect many users to use this as the license list is rather comprehensive already
